src/app/containers/SignupPage/style.scss

```diff
@@ -1,3 +1,5 @@
-// :local {
-
-// }
+.linkDecoration {
+  text-decoration: none;
+  color: #1c83c6;
+}
+  
```

Comment:
Remove this file, the link's style has already been set at the app level.

See `src/app/style.scss`.